drift-transport-netty/src/test/java/com/facebook/drift/transport/netty/client/TestDriftNettyMethodInvoker.java

```diff
@@ -372,6 +372,8 @@ public void testTimeout()
     private static int logNiftyInvocationHandlerOptional(HostAndPort address, List<DriftLogEntry> entries)
     {
         DriftNettyClientConfig config = new DriftNettyClientConfig();
+        config.setTransport(FRAMED);
```

Comment:
is this required?

Reply:
The server it's testing against, in `testProcessor`, is set up explicitly with `TFramedTransport` so we need a `FRAMED` client. I don't see a way to explicitly set up a `HEADER` server.